app/services/event_mailer.rb

```diff
@@ -27,15 +27,34 @@ def listen
     end
 
     @bunny_channel = @bunny_session.channel
+    # Define fanout exchanges which will broadcast
+    # all the messages they receives to all the queues they know
+    retry_exchange = @bunny_channel.fanout('retry.exchange')
```

Comment:
1. let's delete the old queue named postmaster something
2. prefix those exchanges with something like barong.event.mailer
3. rename main mailer queue to barong.event.mailer.main
4. rename retry mailer queue to barong.event.mailer.retry